src/person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name: 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [Optional] When a new person object will be created, is the first method to be call on the new object. and the requested arguments to pass will be name, age and parent_permission. id will not be exposed by requesting it from the person. So I thought it will be nice if you can generate random numbers for the id param. What do you think?